browser/ui/tabs/brave_tab_color_mixer.cc

```diff
@@ -80,15 +80,16 @@ void AddBraveVerticalTabLightThemeColorMixer(ui::ColorProvider* provider,
           {kColorBraveVerticalTabNTBTextColor, SkColorSetRGB(0x6B, 0x70, 0x84)},
           {kColorBraveVerticalTabNTBShortcutTextColor,
            SkColorSetRGB(0x85, 0x89, 0x89)},
+          {kColorBraveSplitViewTileBackground, SkColorSetRGB(0xDA, 0xDF, 0xD1)},
```

Comment:
maybe leo color is not available yet?

Reply:
Yeah, actually the design guide isn't mapped to leo color for this.